qtc_packaging/debian_fremantle/changelog

```diff
@@ -1,3 +1,8 @@
+kodimote (2.0.2-fremantle0) unstable; urgency=low
+  * transition from xbmcremote to kodimote
+
+ -- Michael Zanetti <michael_zanetti@gmx.net>  Sun, 01 Mar 2015 20:00:00 +0200
```

Comment:
Well - I don't know if I'm up to the maintaining stuff - yet. :) But I can change it if you like.
Good point - I just took a shortcut in order get the packaging to work - otherwise it would complain about the package name.
Basically the changelog would need to include all of the library changes, as the UI is still the same. Needs some weeding out I guess. :)